zerver/management/commands/turn_off_digests.py

```diff
@@ -14,27 +14,30 @@ class Command(ZulipBaseCommand):
 
     def add_arguments(self, parser):
         # type: (CommandParser) -> None
-        parser.add_argument('-u', '--users',
-                            dest='users',
-                            type=str,
-                            help='Turn off digests for this comma-separated '
-                                 'list of email addresses.')
-        self.add_realm_args(parser)
+        self.add_realm_args(parser, True)
```

Comment:
Is this change needed?  I can imagine wanting to do this to a list of users from multiple realms.